xlml/utils/metric_test.py

```diff
@@ -346,7 +347,7 @@ def test_add_test_config_metadata(self):
         test_name="test_name",
         set_up_cmds="set_up_cmds",
         run_model_cmds="run_model_cmds",
-        time_out_in_min=60,
+        timeout=datetime.timedelta(minutes=60),
```

Comment:
Could you leave a comment in the pr description of what the intention for using dattime.timedelta?

Reply:
Added in the description.